sagemaker_processing/fairness_and_explainability/text_explainability/text_explainability.ipynb

```diff
@@ -231,10 +231,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/tabular/womens_clothing_ecommerce/Womens_Clothing_E-Commerce_Reviews.csv womens_clothing_reviews_dataset.csv\n",
-    "\n",
-    "# If the above does not work, please uncomment the following lines to download the dataset from s3\n",
-    "#! curl https://sagemaker-sample-files.s3.amazonaws.com/datasets/tabular/womens_clothing_ecommerce/Womens_Clothing_E-Commerce_Reviews.csv > womens_clothing_reviews_dataset.csv"
+    "! curl https://sagemaker-sample-files.s3.amazonaws.com/datasets/tabular/womens_clothing_ecommerce/Womens_Clothing_E-Commerce_Reviews.csv > womens_clothing_reviews_dataset.csv"
```

Comment:
nit: See a space here ```!` `curl```, would this work?

Reply:
I have tested this command and it work in both notebook instances and studio